libs/shared/form-fields/src/lib/InputController/InputController.tsx

```diff
@@ -26,9 +29,37 @@ export const InputController: FC<Props> = ({
   placeholder,
   textarea,
   type = 'text',
+  format,
   onChange,
 }) => {
   const { register } = useFormContext()
+
+  if (format && ['text', 'tel', 'password'].includes(type)) {
+    return (
+      <Controller
+        name={name}
+        render={({ value, onChange }) => (
+          <NumberFormat
+            customInput={Input}
+            id={id}
+            disabled={disabled}
+            name={name}
+            placeholder={placeholder}
+            label={label}
+            type={type as 'text' | 'tel' | 'password'}
```

Comment:
I'm curious to know why the password has to use the numberformat field?

Reply:
☝️

Reply:
@jeremybarbet , @oddsson yeah this does not need to be there, I just copied the type from the NumberFormat component. Removed it.